ci/upstream.yml

```
@@ -28,6 +28,6 @@ dependencies:
   - fsspec
   - pip
   - pip:
-    - icechunk # Installs zarr v3 as dependency
-    # - git+https://github.com/fsspec/kerchunk@main  # kerchunk is currently incompatible with zarr-python v3 (https://github.com/fsspec/kerchunk/pull/516)
-    - imagecodecs-numcodecs==2024.6.1
+      - icechunk==0.1.0a7 # Installs zarr v3 as dependency
```

Better to future-proof it for when the next release of icechunk comes out?
```suggestion
      - icechunk>=0.1.0a7 # Installs zarr v3 as dependency
```